client-next/README.md

```diff
@@ -62,4 +62,4 @@ or add it to a new `.env.development.local` file (this file will be ignored by g
 
 In production mode, the default is to access OTP via the same origin as the client (see `.env`).
 This behavior can also be modified by changing the previously mentioned environment variable at
-build-time.
+build-time..
```

Comment:
Isn't this a typo?
```suggestion
build-time.
```

Reply:
I needed a change in that directory to trigger the CI that's why I added it. I will fix it after the merge.